morebits.js

```diff
@@ -2124,7 +2124,7 @@ Morebits.date.prototype = {
 		level = parseInt(level, 10);
 		level = isNaN(level) ? 2 : level;
 
-		var header = Array(level + 1).join('='); // String.prototype.repeat not supported in IE 11
+		var header = '='.repeat(level);
```

Comment:
Did the +1 get dropped here by mistake?

Reply:
no, it was just hacky code initially: to create a string with n `=` signs, you need an array of n+1 undefined elements and join them with `=`.